src/test/java/step33/CarTest.java

```diff
@@ -0,0 +1,62 @@
+package step33;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CarTest {
+
+    private Car car;
+
+    @BeforeEach
+    void init() {
+        car = new Car(new ArrayList<>());
+    }
+
+    @DisplayName("createCar를 정상적으로 호출했을 경우")
+    @ParameterizedTest
+    @ValueSource(ints = {1,2,3,4,5,6,7,8,9})
+    void createCarTest(int number) {
+        assertDoesNotThrow(() -> car.createCar(number));
+        assertEquals(car.carsSizes().size() , number);
+    }
+
+
+    @DisplayName("createCar를 한번 더 호출해서 exception을 띄울경우")
+    @ParameterizedTest
+    @ValueSource(ints = {1,2,3,4,5,6,7,8,9})
+    void createCarThrowException(int number) {
+        car.createCar(number);
+
+        Assertions.assertThrows(
+                IllegalArgumentException.class,
+                () -> car.createCar(number));
+    }
```

Comment:
TDD 에서 구현보다 실패하는 테스트 코드를 먼저 작성하라고 이야기 합니다.
구현 코드를 먼저 작성하게 되면 구현에 대한 로직이 이미 머리속에 있기 때문에 구현한 대로 테스트 코드를 작성하게 되는 가능성이 생기게 되는데요 의도한 결과대로 성공하는 테스트 코드를 작성할 수는 있지만
내가 생각 하지 못한 부분에서 예외가 발생할 경우에 대한 테스트를 작성 하기가 힘들어 집니다.

해당 테스트 코드가 위와 같은 경우 일 것 같은데요
cars.size() 가 number 보다 크거나 같을 경우 exception이 발생하도록 구현이 되어 있기 때문에 위와 같은 테스트 코드가 작성되었을 것이라고 예상이 됩니다. 
```suggestion
    void createCarThrowException(int number) {
        car.createCar(number);

        Assertions.assertThrows(
                IllegalArgumentException.class,
                () -> car.createCar(number + 1));
    }
```
만약 다음과 같이 테스트 코드를 작성하게 되면 어떻게 될까요? 해당 테스트는 실패하게 되겠네요 😢 
TDD 의 장점에 대해 한번 고민해 보시면 좋을 것 같아요! 😄